zerver/tests/test_events.py

```diff
@@ -1762,7 +1762,7 @@ def test_custom_profile_field_data_events(self) -> None:
             "id": field_id,
             "value": "New value",
         }
-        with self.verify_action() as events:
+        with self.verify_action(num_events=2) as events:
```

Comment:
I think this should also add a check to verify if second event is a message event.